### ISSUE TITLE
Google Chrome and xdg-open fail to open "esr:*" links on Ubuntu Linux

### ISSUE THREAD

**Issue by slyon** [greymass/anchor#1008]
Problem:
When I click an "esr:..." link in Google Chrome on Ubuntu Linux (20.04 LTS), it opens a new blank Chrome window.
Same happens when I call "xdg-open esr:test" from the terminal.
It works fine in Firefox for some reason.

Expected behavior:
Calling `xdg-open esr:test` should open anchor wallet (signing request screen)

Solution:
The problem seems to come from the `anchor-wallet.desktop` file, i.e. this line:
```Exec="/opt/Anchor Wallet/anchor-wallet" %U``` – maybe the space between "Anchor" and "Wallet"?

It works fine if I prepare a symlink `/usr/bin/anchor-wallet -> /opt/Anchor Wallet/anchor-wallet` and change that line to:
```Exec=/usr/bin/anchor-wallet %U```

**Comment by aaroncox** [comment]
Yeah - I'm assuming you're using the AppImage?

I think this may be related to: https://github.com/greymass/anchor/issues/833

You also may be right - perhaps it's the name that's causing the issue. I'll see if simplifying the name for the Linux build (no spaces) resolves the issue 👍

**Comment by slyon** [comment]
> Yeah - I'm assuming you're using the AppImage?

No. I'm using the .deb package actually. Thanks for looking into it!
